src/bias/MetaD.cpp

```diff
@@ -22,26 +22,26 @@
 #include "Bias.h"
 #include "ActionRegister.h"
 #include "core/ActionSet.h"
-#include "tools/Grid.h"
 #include "core/PlumedMain.h"
 #include "core/Atoms.h"
-#include "tools/Exception.h"
 #include "core/FlexibleBin.h"
+#include "tools/Exception.h"
+#include "tools/Grid.h"
 #include "tools/Matrix.h"
+#include "tools/OpenMP.h"
 #include "tools/Random.h"
+#include "tools/File.h"
 #include <string>
 #include <cstring>
-#include "tools/File.h"
 #include <iostream>
 #include <limits>
 #include <ctime>
 #include <memory>
+#include <algorithm>
+#include <numeric>
 
 #define DP2CUTOFF 6.25
 
-using namespace std;
```

Comment:
@carlocamilloni good choice removing this... Code more verbose but way more readable